great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -217,10 +218,24 @@ def _build_parameters(
             variables=variables,
             parameters=parameters,
         )
-        if not (0.0 <= false_positive_rate <= 1.0):
+
+        if false_positive_rate < 0 or false_positive_rate > 1:
             raise ge_exceptions.ProfilerExecutionError(
-                message=f"The confidence level for {self.__class__.__name__} is outside of [0.0, 1.0] closed interval."
+                f"""false_positive_rate must be a positive decimal number between 0 and 1 inclusive [0, 1],
+but {false_positive_rate} was provided."""
+            )
+        elif false_positive_rate <= NP_EPSILON:
+            warnings.warn(
+                f"""You have chosen a false_positive_rate of {false_positive_rate}, which is too close to 0.
+A false_positive_rate of {NP_EPSILON} has been selected instead."""
+            )
+            self._false_positive_rate = NP_EPSILON
+        elif false_positive_rate >= (1 - NP_EPSILON):
+            warnings.warn(
+                f"""You have chosen a false_positive_rate of {false_positive_rate}, which is too close to 1.
+A false_positive_rate of {1-NP_EPSILON} has been selected instead."""
             )
+            self._false_positive_rate = 1 - NP_EPSILON
```

Comment:
@NathanFarmer I believe that these should just be `false_positive_rate` (not `self._ false_positive_rate`).  Do you agree?  Thanks.

Reply:
@alexsherstinsky yes that is correct. But - do we also want to update the object attribute? Otherwise I'm not sure how to assert that `false_positive_rate` was actually changed.

Reply:
@NathanFarmer Only the "non object attribute" is used after we fetch from `variables`.  So the object attribute should not be needed (please confirm via a test that I am not mistaken).  Thanks!

Reply:
The object attribute is only needed for testing. I have removed the object attribute setter and removed the test assertion that checks that `false_positive_rate` was changed.